scrappers/tamil_blasters.py

```diff
@@ -19,7 +19,7 @@
     download_and_save_torrent,
 )
 
-HOMEPAGE = "https://www.1tamilblasters.art"
+HOMEPAGE = "https://www.1tamilblasters.team"
```

Comment:
The previous link is still working and redirecting it to new link. So we can keep as it is for now. Otherwise, changing the link sometime, the new link is only working for few days.

Reply:
Sounds good, will revert.